document/document.go

```diff
@@ -148,3 +149,11 @@ func (d *Document) VisitSynonymFields(visitor index.SynonymFieldVisitor) {
 		}
 	}
 }
+
+func (d *Document) SetSearchable(searchable bool) {
```

Comment:
There is no occasion where the input parameter is false, so change signature to `SetIndexed()`.